test/mocks/genMocks.ts

```diff
@@ -0,0 +1,155 @@
+import "mocha";
+import {
+    BigNumber,
+    ContractTransaction,
+    ethers,
+    Signer,
+} from "ethers";
+import { Block, Provider } from "@ethersproject/providers";
+import { GenSale } from "../../src/contracts";
+
+export const feb23TsSeconds = 1677173138;
+export const feb24TsSeconds = 1677264314;
+
+export type GenMocks = {
+    signer: Partial<Signer>;
+    genSale: Partial<GenSale>;
+    currentSaleConfiguration: [
```

Comment:
Same comment as above, if we can turn this into an interface, type, or maybe even an enum, it would nicer

Reply:
Sale config is now an interface